Examples/Scripts/Python/truth_tracking_kalman.py

```diff
@@ -18,6 +18,7 @@ def runTruthTrackingKalman(
     inputHitsPath: Optional[Path] = None,
     decorators=[],
     reverseFilteringMomThreshold=0 * u.GeV,
+    reverseFilteringCovarianceScaling=1,
```

Comment:
For example, shouldn't this be also 100

Reply:
Hmm so the default in this case is not doing reverse filtering at all (momentum threshold = 0), and have set the scaling value to 1 for consistency. But one can argue that it should always be > 1 of course...

Reply:
OK that makes sense